ui/src/composables/setupValueVariableMap.ts

```diff
@@ -0,0 +1,10 @@
+import { Ref, ref } from "vue";
```

Comment:
feels like this isn't doing a lot except for creating a Map and accessing its set() method, is it worth keeping as a composable?

Reply:
Used in both creator and editor so cleaner code as a composable.